src/App.css

```diff
@@ -96,3 +93,47 @@
   padding-left: 5px;
   color: #060553;
 }
+
+.user_info_container {
+  line-height: 60px;
+  float: right;
+  font-weight: 500;
+}
+
+.dashboard_link {
+  color: #060553;
+  text-decoration-color: #00d4ff;
+  text-decoration-thickness: 2px;
+  margin-right: 30px;
+}
+
+.edit_profile_link {
+  color: #060553;
+  text-decoration-color: #00d4ff;
+  text-decoration-thickness: 2px;
+  margin-right: 90px;
+}
+
+.about_container {
+  height: 610px;
```

Comment:
This class belongs to a div that takes full-screen height, yeah? So why does it have a height of `610px`?.

Reply:
Yes it does, and thank you, i didn't know this. It works

Reply:
@Nkoli Did you check why the `100vh` works?

Reply:
Yes, it's not fixed-ish like using px is and represents a percentage of the height defined as the body height? So 100vh is equivalent to 100% of the initially defined body height. Do i understand this correctly?